### PR TITLE
ABI/codecs implementation (part 2) 

### PR DESCRIPTION
**Pull request by andreibancioiu**
This is part of a series of pull requests.

See: https://github.com/multiversx/mx-sdk-py/pull/32.

This can be reviewed side-by-side with:
 - https://github.com/multiversx/mx-sdk-abi-go/blob/main/abi

### CODE REVIEW

File: multiversx_sdk/abi/fields.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from typing import Any, Dict, List
+
+from multiversx_sdk.abi.interface import SingleValue
+
+
+class Field:
+    def __init__(self, name: str, value: SingleValue) -> None:
+        self.name = name
+        self.value = value
+def encode_fields_nested(fields: List[Field], writer: io.BytesIO):
```
Comment (popenta) [inline multiversx_sdk/abi/fields.py:11]: add empty lines between class and function

File: multiversx_sdk/abi/token_identifier_value.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+
```
Comment (popenta) [inline multiversx_sdk/abi/token_identifier_value.py:2]: unnecessary empty lines but we can live with it :)

**Review by popenta — APPROVED** [review]

**Review by axenteoctavian — DISMISSED** [review]

File: multiversx_sdk/abi/fields.py (hunk 1 above)
Comment (andreibancioiu) [reply multiversx_sdk/abi/fields.py:11]: Fixed in https://github.com/multiversx/mx-sdk-py/pull/49.

File: multiversx_sdk/abi/token_identifier_value.py (hunk 2 above)
Comment (andreibancioiu) [reply multiversx_sdk/abi/token_identifier_value.py:2]: Fixed in https://github.com/multiversx/mx-sdk-py/pull/49.

**Pull request merged**